### PR TITLE
Show config save status (closes #2842)

### LINKED ISSUE #2842: Config: show green check sign in form element if config option request was successfully
Moved from #2053

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
@emanuelschuetze How do you like this behavior?

### CODE REVIEW

**Comment by FinnStutzenstein** [comment]
TODO:
- move check symbol to the right
- symbol on reset

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze Ready for review.

File: openslides/core/static/js/core/site.js [hunk 1]
```diff
@@ -843,26 +843,57 @@ angular.module('OpenSlidesApp.core.site', [
...
+            });
+            var items = _.flatMap(subgroups, function (subgroup) {
+                return subgroup.items;
+            });
+            var configOption = _.find(items, function (_item) {
+                return _item.key === configOption_key;
+            });
+
+            var editor = this;
+            $timeout(function () { // Do this an an angular context
```
Comment (normanjaeckel) [inline openslides/core/static/js/core/site.js]: Typo in the comment. The comment is also not easy to understand. Why do we have a timeout here?

File: openslides/core/static/js/core/site.js [hunk 2]
```diff
@@ -843,26 +843,57 @@ angular.module('OpenSlidesApp.core.site', [
...
-            Config.save(key);
+        $scope.save = function(configOption, value) {
+            Config.get(configOption.key).value = value;
+            Config.save(configOption.key).then(function (success) {
+                configOption.success = true;
+                // fade out the success symbol after 2 seconds.
+                $timeout(function () {
+                    var element = $('#success-field-' + configOption.key);
+                    element.fadeOut(800, function () {
+                        configOption.success = undefined;
```
Comment (normanjaeckel) [inline openslides/core/static/js/core/site.js]: Please use false or void 0 here.

File: openslides/core/static/js/core/site.js (hunk 1 above)
Comment (FinnStutzenstein) [inline openslides/core/static/js/core/site.js]: Done. Yes, this is a normal event. Changes may not directly affect the angular context. See the motion inline factory. The timeouts there do exactly the same.

File: openslides/core/static/js/core/site.js (hunk 2 above)
Comment (FinnStutzenstein) [inline openslides/core/static/js/core/site.js]: Done.

**Comment by emanuelschuetze** [comment]
@normanjaeckel ping

**Pull request merged**